HPXMLtoOpenStudio/measure.rb

```diff
@@ -4192,6 +4289,38 @@ def self.get_foundation_and_walls_top()
     walls_top = foundation_top + 8.0 * @ncfl_ag
     return foundation_top, walls_top
   end
+  
+  def self.get_location_hierarchy(ba_cz_name)
```

Comment:
Since this method is in measure.rb and has a generic name ("get_location_hierarchy"), someone might think it could apply to any object. I suggest making it a static method in the Waterheater class since the logic is specific to water heaters.